config.h

```diff
@@ -529,8 +529,8 @@ __NOTE:__ these definitions are only referenced in this file. Do __NOT__ change!
 //#define DEFAULT_X_ACCELERATION (10.0*60*60) // 10*60*60 mm/min^2 = 10 mm/sec^2
 //#define DEFAULT_Y_ACCELERATION (10.0*60*60) // 10*60*60 mm/min^2 = 10 mm/sec^2
 //#define DEFAULT_Z_ACCELERATION (10.0*60*60) // 10*60*60 mm/min^2 = 10 mm/sec^2
-//#define DEFAULT_X_MAX_TRAVEL 200.0f // mm NOTE: Must be a positive value.
-//#define DEFAULT_Y_MAX_TRAVEL 200.0f // mm NOTE: Must be a positive value.
+#define DEFAULT_X_MAX_TRAVEL 507.0f // mm NOTE: Must be a positive value.
+#define DEFAULT_Y_MAX_TRAVEL 390.0f // mm NOTE: Must be a positive value.
 //#define DEFAULT_Z_MAX_TRAVEL 200.0f // mm NOTE: Must be a positive value.
```

Comment:
Don't we need that for MR Beam 3 ?

Reply:
We need the exact value when the Engineering team finalizes the working area limits for the Mr Beam 3. currently it is set to the limits that are for the existing Mr Beam devices.